Codegen/test/resources/output/TypeCobol/GenerateAsNested/FunDeclareWithExec-PrivateOnly.rdz.cbl

```diff
@@ -0,0 +1,147 @@
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. PGM1.
+
+       DATA DIVISION.
+       Working-STORAGE SECTION.
+
+      *01  mydate1     TYPE Date.
+       01 mydate1.
+           02 YYYY PIC 9(4).
+           02 MM PIC 9(2).
+           02 DD PIC 9(2).
+                                 
+       01  myname1      PIC X(15).
+
+       PROCEDURE DIVISiON.
+
+      *declare procedure check private
+      *   input mydate        TYPE Date
+      *  .
+
+      *declare procedure check private
+      *   input mydate        TYPE Date
+      *   output r1           PIC X(5)
+      *  .
+       
+      *declare procedure checkName private
+      *   input myname        PIC X(15)
+      *  .
+           
+       INIT-LIBRARY.
+            EXIT.
+
+
+      *
+      *declare procedure check private
+      *   input mydate        TYPE Date
+      *  .
+      *_________________________________________________________________
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. a0508f35check.
```

Comment:
This program need to be generated as COMMON.
Every program that is generated as nested either from a public or private procedure must be declared with COMMON.